hail/python/hail/context.py

```diff
@@ -129,7 +129,7 @@ def stop(self):
            _optimizer_iterations=nullable(int))
```

Comment:
A few lines up, you have to change `tmp_dir=str` to `tmp_dir=nullable(str)`

Reply:
Thanks for catching that! Fixed.

I've also realized that `_get_tmpdir` already handles `None`, so this becomes even simpler.